internal/docs/design.md

```diff
@@ -209,3 +220,20 @@ not to do this, for several reasons:
 Overall, massive diffs in the replay files are expected and fine. As part of a
 code change, you may want to check for things like the number of RPCs made to
 identify performance regressions.
+
+## Module Boundaries
+
+With the advent of [Go modules], there are mechanisms to release different parts
+of a repository on different schedules. This permits one API to be in alpha/beta
+(pre-1.0), whereas another API can be stable (1.0 or later).
+
+As of 2018-09-13, Go Cloud as a whole still is not stable enough to call any
+part 1.0 yet. Until this milestone is reached, all of the Go Cloud libraries
+will be placed under a single module. The exceptions are standalone tools like
+[Contribute Bot][] that are part of the project, but not part of the library.
+After 1.0 is release, it is expected that the interfaces of Go Cloud will be
+released as one module and provider implementations will be in a separate
```

Comment:
Wouldn't we want to release different APIs as different modules? I.e., blob might be 1.0 but pubsub might still be 0.x.0.

Reply:
Done.